### PR TITLE
Priestess Alt Title for Chaplain

### PR DESCRIPTION
**Pull request by Catrly**
No idea why my previous PR on this got tangled up.

Adds a Priestess title for chaplain, mostly usable for the tribunal. Requested by Schwann.

### CODE REVIEW

**Comment by Catrly** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Comment by Catrly** [comment]
!review

File: code/game/jobs/job/civilian.dm [hunk 1]
```diff
@@ -276,7 +276,7 @@
 	selection_color = "#90524b"
 	access = list(ACCESS_CHAPEL_OFFICE, ACCESS_MAINT_TUNNELS)
 	minimal_access = list(ACCESS_CHAPEL_OFFICE)
-	alt_titles = list("Presbyter", "Rabbi", "Imam", "Priest", "Shaman", "Counselor", "Keeper", "Shaper")
+	alt_titles = list("Presbyter", "Rabbi", "Imam", "Priest", "Shaman", "Counselor", "Keeper", "Shaper", "Priestess")
```
Comment (Greenjoe12345) [inline code/game/jobs/job/civilian.dm:279]: Maybe Priestess should be put next to Priest?

Comment (Catrly) [reply code/game/jobs/job/civilian.dm]: done

**Pull request merged**